### DISCUSSION TITLE
Is it possible to read files(not included in build) as assets in runtime?

### DISCUSSION THREAD

**Discussion by EsatMan** [stride3d/stride#2372] [General]
Read a .png as a texture for example? Seems that the manual only explains how to use the assets included in the build. Is there a unity-like streamingassets folder by the way?

**Comment by tebjan** [comment]
For basic image formats, you can use the `Texture.Load(..)` method. But generally, the asset compiler is a build step and cannot easily be used at runtime.

**Reply by EsatMan** [reply]
Thanks for reply, I'll look into it.
